projects/angular/src/layout/nav/_header.clarity.scss

```diff
@@ -448,9 +571,56 @@
         header-variables.$clr-header-font-color,
         variables.$clr-use-custom-properties
       );
+
```

Comment:
This comment is reagarding input's text color figma suggest it should be `--cds-global-color-white`. Introducing new token is needed.

Reply:
Done